compiler/rustc_middle/src/ty/vtable.rs

```diff
@@ -60,7 +60,7 @@ impl<'tcx> TyCtxt<'tcx> {
         let ptr_align = tcx.data_layout.pointer_align.abi;
 
         let vtable_size = ptr_size * u64::try_from(vtable_entries.len()).unwrap();
-        let mut vtable = Allocation::uninit(vtable_size, ptr_align)?;
+        let mut vtable = Allocation::uninit(vtable_size, ptr_align, true)?;
```

Comment:
```suggestion
        let mut vtable = Allocation::uninit(vtable_size, ptr_align, /* panic_on_fail */ true)?;
```
Is this the only `?` in this function, so we could `unwrap` right away and not return `Result`?